### PR TITLE
add required value to parameter describer

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@csrwng ptal.

**Comment by bparees** [comment]
[test]

File: pkg/cmd/cli/describe/describer.go [hunk 1]
```diff
@@ -689,6 +689,7 @@ func (d *TemplateDescriber) DescribeParameters(params []templateapi.Parameter, o
 	for _, p := range params {
 		formatString(out, indent+"Name", p.Name)
 		formatString(out, indent+"Description", p.Description)
+		formatString(out, indent+"Required", p.Required)
```
Comment (miminar) [inline pkg/cmd/cli/describe/describer.go]: It seems that `formatString` can't handle booleans. I get:

    $ oc describe template postgresql-persistent
    Name:           postgresql-persistent
    ...
    Parameters:              
        Name:               DATABASE_SERVICE_NAME
        Description:        Database service name
        Required:           %!!(MISSING)s(bool=true)
        Value:              postgresql
    ...

Comment (csrwng) [inline pkg/cmd/cli/describe/describer.go]: @bparees here's a pull with a suggested fix https://github.com/bparees/origin/pull/5

Comment (bparees) [inline pkg/cmd/cli/describe/describer.go]: thanks, will merge this after your PR merges.

**Comment by bparees** [comment]
[merge]

**Comment by bparees** [comment]
flake. [merge]

**Comment by liggitt** [comment]
@fabianofranz:
```
1)  authenticated e2e-user with test project when adding to project should create from source
  - Error: Wait timed out after 5116ms
      at /data/src/github.com/openshift/origin/assets/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver/promise.js:1425:29
```

**Comment by bparees** [comment]
definition of insanity: doing the same thing over and over again and expecting different results... [merge]

**Comment by liggitt** [comment]
with any luck, https://github.com/openshift/origin/pull/4152 will get a clean run and resolve the UI test timeouts

**Comment by fabianofranz** [comment]
https://github.com/openshift/origin/pull/4161 disables some complex web console test scenarios that are making the phantomjs browser to flake. Hopefully we can get a clear run after that gets in. Sorry about that.

**Comment by fabianofranz** [comment]
re[merge]

**Pull request merged**